### PR TITLE
Update 04-InHisGrace76.md

### PR DESCRIPTION
**Pull request by InHisGrace76**
Updated Caption Area.

### CODE REVIEW

File: _slides/04-InHisGrace76.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
----
-
-Caption here
-
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Hope This Works! 😁
```
Comment (InHisGrace76) [inline _slides/04-InHisGrace76.md:6]: # Yep it Works right Here ... 👍🏽
#17 
# Cool

Comment (InHisGrace76) [reply _slides/04-InHisGrace76.md:6]: On to the next step

Comment (samkrishnan) [inline _slides/04-InHisGrace76.md:6]: This is the changed line i believe

**Review by samkrishnan — COMMENTED** [review]
This is good. but it does not give me an "approve" option b/c it is approved.

**Review by samkrishnan — APPROVED** [review]

**Pull request merged**